hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -742,6 +743,53 @@ private HapiSpec directTransferWorksForERC20() {
                                                         .withErcFungibleTransferStatus(true)))))));
     }
 
+    HapiSpec transferHbarsViaEip2390TxSuccessfully() {
```

Comment:
Can we add a HapiSpec for make a contract function call using 2930 tx type also.
If there's already one present then it's fine

Reply:
Added for both contract function `transfer` and contract call `totalSupply`. Feel free to ping me if we need more coverage :rocket:.